tests/longevity/longevity_test.go

```diff
@@ -180,6 +183,7 @@ func testTrigger(wg *sync.WaitGroup,
 	for {
 		// if timeout is 0, run indefinitely
 		if timeout != 0 && int(time.Since(start).Seconds()) > timeout {
+			logrus.Infof("Longevity Tests timed out with timeout %d  minutes", minRunTime)
```

Comment:
There should be more to be done here. We should come out with timeout, for an example for 1TB pool size it should not be greater than 6 hours and based on this we can calculate the timeout for rebalance. 

This would be useful if pool rebalance operation stuck forever and we should mark test fail. The issue we observed pool rebalance got stuck.

Reply:
Actually this should monitor pool rebalance progress percentage if percentage doesn't change for long time, we should mark test failed.

Reply:
Hi Smit,

this change is for giving user an option to timeout Longevity after running certain time as Longevity in long running test
Ill work on pool rebalnace in another PR